roles/docker-garbage-collection/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: "install dateutil library"
+  pip:
+    name: python-dateutil
+    state: latest
+
+- name: "create docker_garbage_collection directory"
+  file:
+    path: "{{garbage_home}}"
+    state: "directory"
+    owner: "{{garbage_user}}"
+    group: "{{garbage_group}}"
+    mode: "0754"
+
+- name: "create image backup directory"
+  file:
+    path: "{{backup_images}}"
+    state: "directory"
+    owner: "{{garbage_user}}"
+    group: "{{garbage_group}}"
+    mode: "0754"
+
+- name: "create container backup directory"
+  file:
+    path: "{{backup_containers}}"
+    state: "directory"
+    owner: "{{garbage_user}}"
+    group: "{{garbage_group}}"
+    mode: "0754"
+
```

Comment:
The above three tasks should be able to combine into one task. For example: https://github.com/osu-mist/ansible-roles/blob/master/roles/api-environment/tasks/main.yml#L16-L28